app/views/handlers/list_action.py

```diff
@@ -15,9 +15,15 @@ def parent_location(self):
             block_id=self.rendered_block["parent_id"],
         )
 
+    @property
+    def section_routing_path(self):
+        return self.router.section_routing_path(
+            section_id=self._current_location.section_id
+        )
```

Comment:
This method shouldn't be needed, can we override the `_get_routing_path` method from `BlockHandler` and use the parent location rather than current location?